src/components/Invoice.js

```diff
@@ -0,0 +1,104 @@
+import { Typography, Paper, Stack, Chip, Box } from '@mui/material';
+
+function Invoice(props) {
+
+  const COLORS = {
+    ISSUED: '#64CEFB',
+    PAID: '#7DBE00',
+    OVERDUE: '#CF4520',
+  };
+   const TEXT = {
+    ISSUED: 'Due on',
+    PAID:' Paid on',
+    OVERDUE:'Was due on',
+  }
+
+  const MONTHS = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+
+  const getMonth = MONTHS[props.due.split("/")[1] - 1];
```

Comment:
The `get` prefix in this name makes me think of a function, but this is not a function, it's just a variable.
```suggestion
  const month = MONTHS[props.due.split("/")[1] - 1];
```